yb-voyager/cmd/exportSchema.go

```diff
@@ -79,6 +81,15 @@ func exportSchema() {
 
 	CreateMigrationProjectIfNotExists(source.DBType, exportDir)
 	source.DB().ExportSchema(exportDir)
+	schemaList := utils.GetSchemaObjectList(source.DBType)
+	for _, objectType := range schemaList {
+		if !slices.Contains([]string{"VIEW", "MVIEW"}, objectType) {
+			continue
+		}
```

Comment:
Drop this condition. Our processImportDirectives() implementation will work even if a file doesn't have any import directives.